pages/about/index.md

```diff
@@ -28,10 +49,69 @@ Would you like to get on the list and support us financially? Check out the [spo
 
 Would you like to help us with the maintainance? Contact us on [slack](https://www.asyncapi.com/slack-invite/).
 
-## History
+## Brands and companies using AsyncAPI
+The total number of companies and projects using AsyncAPI as well as their identity is impossible to know. So far, we are aware that the following ones operate with it.
+- [**Slack**](https://slack.com/signin#/signin): Business Communication Platform that permits to operate with multiple channels (public and private). It currently has more than 10 million daily active users worldwide. It was recently acquired by Salesforce.
+- [**Adidas**](https://www.adidas.com/us): Company dedicated to the manufacture of sports equipment and fashion products. It is the largest sportswear manufacturer in Europe, second largest in the world. This company has more than 150.000 sales outlets around the world.
+- [**Salesforce**](https://login.salesforce.com/?locale=es): Business consultancy company. It provides (CRM) service and other services. It manages from small businesses to FORTUNE 500 companies and it has more than 150,000 customers.
+- [**IBM**](https://www.ibm.com): Company that manufactures and markets hardware and software. It has operations in over 170 countries and provides hosting and consulting services in many areas.
+- [**SAP**](https://www.sap.com/index.html): Company dedicated to the design of computer products for business management. Develops business software to manage operations and business-to-customer relationships. It’s a large company with 100,330 employees.
+- [**IQVIA**](https://www.iqvia.com/"): Company providing services for the combined health information technology and clinical research industries. It employs more than 58.000 people in over 100 countries
 
-This project started at Hitch (a.k.a. API Changelog). We are in love with API machine-readable documentation formats (OpenAPI, RAML, etc.) and always encouraged our customers to use a machine-readable definition for their APIs because it allows them to get extra benefits. From documentation and code generation to setup their own [API Assistant](http://apiassistant.com).
+## Open governance model
```

Comment:
I think we should first put it before sections about maintainers and contributors, after history